### ISSUE TITLE
Double NerMiner

### ISSUE THREAD

**Issue by pmiguelr** [BitMaker-hub/NerdMiner_v2#357]
How I can build one double with 2 S3 displays?
I need to by it soldered with headers?

**Comment by sany3001** [comment]
Something like this?

https://www.tindie.com/products/electronictrik/quatrominer-for-nerdminer/

**Comment by pmiguelr** [comment]
Yes. Both for two?And in code. What I need to do?Where I can find shield for two?Board is with header soldered?Enviado de Outlook para AndroidDe: sany3001 ***@***.***>Enviado: domingo, março 24, 2024 7:19:05 da tardePara: BitMaker-hub/NerdMiner_v2 ***@***.***>Cc: pmiguelr ***@***.***>; Author ***@***.***>Assunto: Re: [BitMaker-hub/NerdMiner_v2] Double NerdMiner (Issue #357)
Something like this?
https://www.tindie.com/products/electronictrik/quatrominer-for-nerdminer/

**Comment by sany3001** [comment]
This is just HW. No need to do any changes in SW. All devices are running independently.
If you setup same BTC address in all your devices you can see statistic on device like ESP32-2432S028R. See below.

[
![IMG_7080](https://github.com/BitMaker-hub/NerdMiner_v2/assets/31476550/95261897-f782-4e93-bded-a2de7e9c819a)
](url)

![Uploading IMG_7081.jpg…]()
